src/topology/homotopy/product.lean

```diff
@@ -44,6 +46,15 @@ of products.
 
 - `path.homotopic.prod` The product of two path classes.
 
+## Fundamental groupoid preserves products
```

Comment:
There is abstract machinery in `category_theory` about functors that `preserves_products`. It would be good to hook up to that machinery. Maybe you can even reuse some of it for the proofs below.
The trouble is that the categorical machinery doesn't use the "canonical construction" for products in `Top` or products of groupoids. It just chooses some object with the guarantee that it satisfies the universal property. So a bit of glue will be needed. At the same time, there is a lot of categorical machinery that will be unlocked, once that glue is provided.